src/Header.tsx

```diff
@@ -1,19 +1,22 @@
 import { Link } from 'react-router-dom';
+import { BlueprintIcon, StyledHeader, StyledHeaderList, StyledHeaderLink } from './content/json/header-assets';
 
 interface HeaderProps {
   links: string[];
 }
 
 const Header = ({ links }: HeaderProps) => {
   return (
```

Comment:
Since this has no state or anything, you could re-write this to be a one-line function.

e.g. instead of `const Header =  ({ links }: HeaderProps) => { return (JSX stuff); }`
you can write: `const Header =  ({ links }: HeaderProps) => (JSX stuff);`